src/main/java/com/github/igotyou/FactoryMod/utility/FileHandler.java

```diff
@@ -300,6 +302,16 @@ private void loadFromFile(File f, Map<String, IFactoryEgg> eggs) {
 					}
 				}
 				fac.setAutoSelect(autoSelect);
+				Chest chestBlock = (Chest) (fac.getChest().getState());
```

Comment:
This would force load every factory on server startup, which for multiple thousand factories might take several minutes.

Reply:
Damn, I should have caught that. Removed changes to FileHandler.java